ai-data/generative-apis/concepts.mdx

```diff
@@ -36,6 +36,12 @@ Parameters are settings that control the behavior and performance of generative
 
 The inter-token latency (ITL) corresponds to the average time elapsed between two generated tokens. It is usually expressed in milliseconds.
 
+## JSON mode
+
+JSON mode allows you to guide the language model in outputtiong well-structured JSON data that matches a provided schema.
+To activate JSON mode, provide the `response_format` parameter in with `{"type": "json_object"}`. It is possible to define a JSON schema using the `schema` property of `response_format`, ensuring the generated output meets your requirements.
```

Comment:
Should we remove either the `in` or `with` in the part 'provide the response_format parameter **in with**...`?

Reply:
I think we can go for `provide the `response_format` parameter with `{"type": "json_object"}`.` wdyt @jcirinosclwy ?